src/Tools/Common/Commands/Utils.cs

```diff
@@ -27,15 +28,14 @@ public static int FindProcessIdWithName(string name)
                 {
                     if (commonId != -1)
                     {
-                        Console.WriteLine("There are more than one active processes with the given name: {0}", name);
-                        return -1;
+                        throw new CommandLineErrorException($"There are more than one active processes with the given name: {name}");
```

Comment:
Maybe CommandLineErrorException is a little misplaced in a utility function, data might input parameter might come for command line or some other area. We should create a specific exception for these errors or use some more generic exception type.

Reply:
I think its reasonable to use CommandLineErrorException when the code is sufficiently constrained that we know its going to called by the command handlers and we know we want any errors to get shown on the console afterwards. I think anything under Commands/Utils probably fits that condition. IMO creating more generic Exceptions will just lead to every caller changing a 1 line call into an 8 line try/catch to convert the exception type back:
```
try
{
   UtilityFunction();
}
catch(SomeOtherExceptionType e)
{
    throw CommandLineErrorException(some_message);
}
```